globus_cli/commands/login.py

```diff
@@ -13,6 +13,23 @@
     internal_auth_client, write_option)
 
 
+_LOGIN_EPILOG = ("""\
+
+SUCCESS
+You have logged in to the Globus CLI as {}
+
+A set of valid tokens for accessing Globus have been saved for you in
+~/.globus.cfg and are valid indefinitely unless revoked.
```

Comment:
Do we need to tell people where the tokens have been saved? It seems like we're just making it easy for someone to mess around with their config file directly. Not mentioning this also means we don't have to account for Windows not knowing what `~` means.